File: bootstrap/archived_runa_0.0.7.5+/v0.0.8.2/benchmarks/rust/stringproc.rs
fn string_length(s: &str) -> usize {
    s.len()
}

fn copy_string(src: &str) -> String {
    src.to_string()
}

fn main() {
    let iterations = 10000;
    let base_str = "Hello";
    let mut total_len = 0;

    for _ in 0..iterations {
        let result_str = copy_string(base_str);
        let len = string_length(&result_str);
        total_len += len;
    }

    std::process::exit(0);
}

File: bootstrap/v0.0.8.4.5.1/benchmarks/java/Stringproc.java
public class Stringproc {
    static int stringLength(String str) {
        return str.length();
    }

    static String copyString(String src) {
        return new String(src);
    }

    public static void main(String[] args) {
        int iterations = 10000;
        String baseStr = "Hello";
        int totalLen = 0;

        for (int i = 0; i < iterations; i++) {
            String resultStr = copyString(baseStr);
            int len = stringLength(resultStr);
            totalLen += len;
        }

        System.exit(0);
    }
}
